spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt

```diff
@@ -1472,6 +1473,20 @@ private fun ColorPreview(
                     ColorItem(SparkTheme.colors.neutralContainer, "neutral Container")
                 }
             }
+            Column {
+                Row {
+                    ColorItem(SparkTheme.colors.background, "background")
+                    ColorItem(SparkTheme.colors.backgroundVariant, "backgroundVariant")
+                }
+                Row {
+                    ColorItem(SparkTheme.colors.surface, "surface")
+                    ColorItem(SparkTheme.colors.surfaceInverse, "surface inverse")
+                }
+                Row {
+                    ColorItem(SparkTheme.colors.outline, "outline")
+                    ColorItem(SparkTheme.colors.outlineHigh, "outline High")
+                }
+            }
```

Comment:
For instance here, sometimes we have the exact same string, but some other time we have split with spaces